src/components/flex/Flex.stories.mdx

```diff
@@ -0,0 +1,428 @@
+      description: '(Responsive)',
+      table: {
+        type: {
+          summary: formatTags(Object.values(FLEX_ALIGNMENT_VALUES)),
+        },
+      },
+      control: {
+        type: 'select',
+        options: Object.values(FLEX_ALIGNMENT_VALUES),
+      },
+    },
+    alignItems: {
+      description: '(Responsive)',
+      table: {
+        type: {
+          summary: formatTags(Object.values(FLEX_ALIGNMENT_VALUES)),
+        },
+      },
+      control: {
+        type: 'select',
+        options: Object.values(FLEX_ALIGNMENT_VALUES),
+      },
+    },
+    alignSelf: {
+      description: '(Responsive)',
+      table: {
+        type: {
+          summary: formatTags(Object.values(FLEX_ALIGNMENT_VALUES)),
+        },
+      },
+      control: {
+        type: 'select',
+        options: Object.values(FLEX_ALIGNMENT_VALUES),
+      },
+    },
+    inlineFlex: {
+      description: '(Responsive)',
+      table: {
+        type: {
+          summary: 'boolean',
+        },
+      },
+      control: {
+        type: 'boolean',
+      },
+    },
+    wrap: {
+      description: '(Responsive)',
+      table: {
+        type: {
+          summary: 'boolean',
+        },
+      },
+      control: {
+        type: 'boolean',
+      },
+    },
+    wrapReverse: {
+      description: '(Responsive)',
+      table: {
+        type: {
+          summary: 'boolean',
+        },
+      },
+      control: {
+        type: 'boolean',
+      },
+    },
+    margin: {
+      description: '(Responsive)',
+      table: {
+        type: {
+          summary: formatTags(Object.values(FLEX_MARGINS)),
+        },
+      },
+      control: {
+        type: 'select',
+        options: Object.values(FLEX_MARGINS),
+      },
+    },
+    marginTop: {
+      description: '(Responsive)',
+      table: {
+        type: {
+          summary: formatTags(Object.values(FLEX_MARGINS)),
+        },
+      },
+      control: {
+        type: 'select',
+        options: Object.values(FLEX_MARGINS),
+      },
+    },
+    marginRight: {
+      description: '(Responsive)',
+      table: {
+        type: {
+          summary: formatTags(Object.values(FLEX_MARGINS)),
+        },
+      },
+      control: {
+        type: 'select',
+        options: Object.values(FLEX_MARGINS),
+      },
+    },
+    marginBottom: {
+      description: '(Responsive)',
+      table: {
+        type: {
+          summary: formatTags(Object.values(FLEX_MARGINS)),
+        },
+      },
+      control: {
+        type: 'select',
+        options: Object.values(FLEX_MARGINS),
+      },
+    },
+    marginLeft: {
+      description: '(Responsive)',
+      table: {
+        type: {
+          summary: formatTags(Object.values(FLEX_MARGINS)),
+        },
+      },
+      control: {
+        type: 'select',
+        options: Object.values(FLEX_MARGINS),
+      },
+    },
+  }}
+/>
+
+# Flex
+
+- [Stories](#stories)
+- [Accesibility](#accessibility)
+- [Responsive props](#responsive-props)
```

Comment:
Responsive props section is before Accessibility

Reply:
@katarzynatobis Done